README.md

```diff
@@ -135,6 +135,13 @@ More detailed documentation is available here: https://oneapi-src.github.io/unif
 
 ### Memory providers
 
+#### Coarse Provider
+
+A memory provider that can manage a memory (i.e. handle the alloc() and free() ops)
+of a given pre-allocated buffer or of an additional upstream provider (e.g. OS Memory Provider
```

Comment:
based on this description I still don't know why should I use this provider with another provider...? could you pls make it more clear?

Reply:
Changed.
Done